### PR TITLE
Updates from rspec-dev (2020-11-16)

### PR DESCRIPTION
**Pull request by JonRowe**
These are some updates, generated from rspec-dev's rake tasks.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -52,3 +43,4 @@ branches:
   only:
     - main
     - /^\d+-\d+-maintenance$/
+    - /^\d+-\d+-dev$/
```
Comment (pirj) [inline .travis.yml:46]: :heart:

**Review by pirj — APPROVED** [review]
Is it possible to only run one build? Either for the branch (`main`, `*-maintenance`, `*-dev`) or for the PR?

**Comment by JonRowe** [comment]
> Is it possible to only run one build? Either for the branch (main, *-maintenance, *-dev) or for the PR?

Don't know, my standard knowledge says that you need `pr` so that, er pull requests are built, but you also need `push` so that branches pushed up by us run.

**Comment by JonRowe** [comment]
We might be able to limit push to `main` and maintenance branches?

**Comment by JonRowe** [comment]
This sometimes happens with Travis as well by the way, it's just we don't see it

**Comment by pirj** [comment]
> This sometimes happens with Travis as well by the way, it's just we don't see it

We found a reason for that with Benoit, it's that removed `branches` setting for `rspec-rails`. Doesn't happen with other repos.

**Comment by JonRowe** [comment]
> We found a reason for that with Benoit, it's that removed branches setting for rspec-rails. Doesn't happen with other repos.
That branches option was deliberately limiting builds on rspec-rails becauase of the load it causes.

**Comment by pirj** [comment]
> That branches option was deliberately limiting builds on rspec-rails 

`branches` option is [not only in `rspec-rails`](https://github.com/rspec/rspec-support/blob/9a2a668c83b858901052329518da414c299405ac/.travis.yml#L51):
```yaml
branches:
  only:
    - main
    - /^\d+-\d+-maintenance$/
```

Frankly, we have everywhere except `rspec-rails`, resulting in exactly the opposite effect :laughing:

**Pull request merged**